### ISSUE TITLE
"An unexpected error has occurred" - on newly opened design when selecting a layer

### ISSUE THREAD

**Issue by andrew-mykhalchuk** [avocode/avocode#2208]
This error occurred when I opened a PSD design and selected a layer. Avocado didn't crash - just the modal pope with this error message. In message body it says:
The application has encountered an error.
        If you see a pattern or have any information
        which might be of use for us.
 Please report this issue.

I'm currently on the latest version but prior to this message I had a message that new version is available so maybe this is linked to Avocode update process somehow as it didn't occur earlier when I was inspecting this same PSD.

---
* **Version**: 3.6.0
* **Sketch plugin version**: 51.2.0
* **Web/Desktop**: Desktop
* **Platform**: darwin
* **Avocode UID**: 67766
---
```
TypeError: Cannot read property 'set' of undefined
    at file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:1:4052319
    at updateInDeepMap (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/immutable/dist/immutable.js:1973:22)
    at updateInDeepMap (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/immutable/dist/immutable.js:1982:23)
    at updateInDeepMap (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/immutable/dist/immutable.js:1982:23)
    at Map.updateIn (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/immutable/dist/immutable.js:1280:26)
    at file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:1:4052283
    at /Applications/Avocode.app/Contents/Resources/app.asar/node_modules/immutable/dist/immutable.js:3907:58
    at /Applications/Avocode.app/Contents/Resources/app.asar/node_modules/immutable/dist/immutable.js:2701:43
    at List.__iterate (/
```

**Comment by crashmajstr** [comment]
Hi @mehocode  , we're currently working on a fix, I am sorry for the inconvenience, we'll let you know once it's fixed. 
Closing with respect to #2202 
Best,
Josef

**Comment by crashmajstr** [comment]
Hi, just wanna inform you that we’ll release the fix in an upcoming hour or so. It’ll be version 3.6.1.
For now, you can just download and use our older stable build 3.5.6 

Download links
Mac: https://we.tl/t-bS4jyZufpT
Win: https://we.tl/t-12nDKmpafd
Linux: https://we.tl/t-krrbw7ZRvB

I’ll let you know once the fix is out. Sorry for the inconvenience.

**Comment by crashmajstr** [comment]
Hi, just wanna inform you that we’ve just released the new version of Avocode 3.6.1 containing the fix for this issue. 
Please, update your desktop app, if you use the web app the new version is already there.
Hope everything will be ok from now on.
I am very sorry for the inconvenience. 
On behalf of Avocode,
Josef Kettner

**Comment by andrew-mykhalchuk** [comment]
Hi @crashmajstr - I can confirm this is now fixed. I'm on v3.6.2 already. Appreciate blazing fast updates!

Thank you
